R/dv8.R

```diff
@@ -0,0 +1,66 @@
+#' Convert from sdsm.json to a dependency matrix in sdsm.dv8-dsm file
+#'
+#' @param sdsm_json_path path to sdsm.json
+#' @param dv8_dsm_save_path path to save dv8-dsm file
+#' @export
+dv8_dsmj_to_dsmb <- function(sdsm_json_path, dv8_dsm_save_path){
```

Comment:
Hi, 

Please see my PR comment notes to Leilani, as requested by e-mail: https://github.com/sailuh/kaiaulu/pull/171/files#r1151621045 

To be consistent, the naming of the parameters here would be dsmj_path and dsmb_path